utils/oscap.8

```diff
@@ -191,6 +191,11 @@ Do not validate input/output files.
 Do not validate digital signatures in digitally signed SCAP source data streams.
 .RE
 .TP
+\fB\-\-enforce-signature\fR
```

Comment:
Please add it also to `dist/bash_completion.d/oscap`.

Reply:
Damn, always forgetting it. Thanks.